dc-cudami-server/dc-cudami-server-backend-jdbi/src/main/java/de/digitalcollections/cudami/server/backend/impl/jdbi/identifiable/entity/EntityRepositoryImpl.java

```diff
@@ -74,12 +73,9 @@ public static String getSqlUpdateFieldValues() {
 
   @Autowired
   private EntityRepositoryImpl(
-      Jdbi dbi,
-      IdentifierRepository identifierRepository,
-      FileResourceMetadataRepositoryImpl fileResourceMetadataRepositoryImpl) {
+      Jdbi dbi, FileResourceMetadataRepositoryImpl fileResourceMetadataRepositoryImpl) {
```

Comment:
Hier wird nach wie vor ein Repository verwendet, gehört wohl eher in den Service, oder?

Reply:
Wird nur für Select gebraucht. Könnte also hier bleiben.

Reply:
Ok :+1: